modules/mssql/outputs.tf

```diff
@@ -85,3 +85,11 @@ output "primary" {
   description = "The `google_sql_database_instance` resource representing the primary instance"
   sensitive   = true
 }
+
+output "apphub_service_uri" {
+  value = {
+    service_uri = "//cloudsql.googleapis.com/projects/${var.project_id}/instances/${var.name}"
```

Comment:
Rather than interpolating variable names let's extract from computed attribute like self_link

https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/sql_database_instance#self_link

Reply:
I was using `self_link` initially but @q2w pointed out that the output is not in CAIS format (its `www.googleapis.com/sql/...` instead of `cloudsql.googleapis.com/...`). 

I'm not actually familiar with how these parameters are going to be used in Apphub. Is there a difference between these two representations?

Reply:
@tjy9206 I assume Bharat's point is to extract `projects/${var.project_id}/instances/${var.name}` from self_link! Here is an example of self_link `https://sqladmin.googleapis.com/sql/v1beta4/projects/abhiwa-test-30112023/instances/database-instance-2`

Reply:
Thank you both! Modified accordingly.